external-crates/move/crates/move-package-alt/src/compatibility/legacy_parser.rs

```diff
@@ -752,7 +752,10 @@ fn get_manifest_address_info(
     if published_at.is_some()
         && (original_id.is_none() || original_id.is_some_and(|id| id == AccountAddress::ZERO))
     {
-        bail!("If `published-at` is defined in Move.toml, `original-id` must also be defined.");
+        bail!(
+            "If `published-at` is defined in Move.toml, the [addresses] section must contain a \
```

Comment:
How do we tell which named address to use? Do we know that here? Perhaps the message should include the name for the address.